src/Aquifer.Jobs/Managers/SendResourceAssignmentNotificationsManager.cs

```diff
@@ -68,8 +82,10 @@ public async Task RunAsync([TimerTrigger(CronSchedules.EveryTenMinutes)] TimerIn
                     {
                         [EmailMessagePublisher.DynamicTemplateDataSubjectPropertyName] = "Aquifer Notification: Resources Assigned",
                         ["aquiferAdminBaseUri"] = _configurationOptions.Value.AquiferAdminBaseUri,
-                        ["resourceCount"] = userGrouping.Count(),
+                        ["resourceCount"] = countOfResourcesAssignedToUser,
+                        ["additionalResourceCount"] = Math.Max(countOfResourcesAssignedToUser - _maxNumberOfResourcesToDisplayInNotificationContent, 0),
                         ["parentResources"] = userGrouping
+                            .Take(_maxNumberOfResourcesToDisplayInNotificationContent)
```

Comment:
We could potentially improve this to spread out resources displayed in order to include one for each parent resource but the number of emails where that would be valuable are likely very small.